src/Makefile.am

```diff
@@ -758,7 +759,7 @@ squid.conf.default squid.conf.documented: cf_parser.cci
 cf_parser.cci: cf.data cf_gen$(EXEEXT)
 	./cf_gen$(EXEEXT) cf.data $(srcdir)/cf.data.depend
 
-# The cf_gen_defines.cci is auto-generated and does not exist when the 
+# The cf_gen_defines.cci is auto-generated and does not exist when the
```

Comment:
Unrelated formatting change. Please revert.
```suggestion
# The cf_gen_defines.cci is auto-generated and does not exist when the 
```

Reply:
ok. I'll run a sourceformat before merging just to be on the safe side